internal/utils/copy_from_image.go

```diff
@@ -38,6 +39,70 @@ func CopyFromImage(image, srcInImage, destOnHost string) error {
 	if err != nil {
 		return fmt.Errorf("copy from image: %s: %w", stderr.String(), err)
 	}
+	excludeFiles := []string{".gitkeep", ".ini", ".env", ".DS_Store"}
+	excludeDirs := []string{".git", ".zeabur"}
+	err = deleteFilesInDirectory(excludeFiles, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete files in directory: %w", err)
+	}
+	err = deleteDirectoriesInDirectory(excludeDirs, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete directories in directory: %w", err)
+	}
+
+	return nil
+}
+
+func deleteFilesInDirectory(deleteFiles []string, path string) error {
+	// walk through the directory
+	err := filepath.Walk(path, func(filePath string, fileInfo os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !fileInfo.IsDir() {
+			fileName := fileInfo.Name()
+
+			for _, targetFile := range deleteFiles {
+				if fileName == targetFile {
```

Comment:
Use [`strings.EqualFold`](https://pkg.go.dev/strings@go1.21.3#EqualFold) for case-insensitive filesystem.